docs/api/genomicRdd.rst

```diff
@@ -152,3 +152,110 @@ Similar to ``transform``/``transformDataset``, there exists a
 ``transmuteDataset`` function that enables transformations between
 ``GenomicRDD``\ s of different types.
 
+Using partitioned Parquet to speed up range based queries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+GenomicRDDs of types ``AlignmentRecordRDD``, ``GenotypeRDD``, 
+``VariantRDD``, and ``NucleotideFragmentContigRDD`` can be written as Parquet 
+using a Hive-style hierarchical directory scheme that is based on contig and 
+genomic position.  This partitioning reduces the latency of genomic range
+queries against these datasets, which is particularly important for interactive
+applications such as a genomic browser backed by an ADAM dataset.
+
+The genomicRDD function 
+``GenomicRDD.filterByOverlappingRegions(queryRegionsList)`` builds a Spark SQL 
+query that uses this paritioning scheme. This can reduce latencies by more 
+than 20x when repeatedly querying a datset with genomic range filters.   
+On a high coverage alignment dataset, this partitioning strategy improved
+latency from 1-2 minutes to 1-3 seconds when looking up genomics ranges.
+
+**Saving partitioned parquet files to disk**  
+
+A ``GenomicRDD`` can be written to disk as a paritioned Parquet dataset with the
+``GenomicRDD`` function ``saveAsParitionedParquet``. The optional
+``partitionSize`` parameter  defines the width in base pairs of the partitions
+within each contig.
+
+.. code:: scala
+ data.saveAsPartitionedParquet("dataset1.adam", partitionSize = 2000000)
+
+A partitioned dataset can also be created from an input Parquet or SAM/BAM/CRAM
+file using the ADAM ``transformAlignments`` CLI, or Parquet/VCF files using the
+ADAM ``transformGenotypes`` CLI.
+
+**Loading partitioned parquet files**
+
+A GenomicRDD can be loaded from a partitioned {arquet dataset using the
+ADAMContext functions ``loadPartitionedParquetDataset`` or by using the
+``--isBinned`` options in the CLI.
```

Comment:
what is `--isBinned`?

Reply:
fixed, changed to the "-partitioned_by_start_pos" option

Reply:
actually, fixed by getting rid of reference to loading from CLI here.